src/systems/camera.js

```diff
@@ -10,9 +10,23 @@ var DEFAULT_CAMERA_ATTR = 'data-aframe-default-camera';
 module.exports.System = registerSystem('camera', {
   init: function () {
     this.activeCameraEl = null;
+    this.cameraEls = [];
     this.setupDefaultCamera();
   },
 
+  addCamera: function (el) {

```

Comment:
Just going to double-note here to call it `cameraEl` when it's an entity, and `camera` when it's an object3D.